microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,171 @@
 ---
-title: Windowed vs. visual hosting of WebView2
-description: Deciding whether to have your app use windowed hosting vs. visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
+title: Windowed vs. Visual hosting of WebView2
+description: Deciding whether to have your app use Windowed, Window to Visual, or Visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 10/24/2022
+ms.date: 09/23/2024
 ---
-# Windowed vs. visual hosting of WebView2
+# Windowed vs. Visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+There are three options for hosting the Microsoft Edge WebView2 control in your app:
+* The Windowed hosting mode.
+* The Window to Visual hosting mode.
+* The Visual hosting mode.
 
-If you use windowed hosting, which is a good starting point for most apps, you don't need to read this article.  If you want to provide a more custom user experience (UX), and want to use visual hosting, read this article.
+You don't need to read this article if you use Windowed hosting in most scenarios.  Windowed hosting is a good starting point for most apps.  Read this article:
+* If you are using Windowed hosting in uncommon scenarios and are experiencing persistent issues with DPI and scaling.  In this case, consider Window to Visual hosting.
+* If you want to provide a more custom user experience (UX).  In this case, consider Visual hosting.
+
+The different approaches for hosting the WebView2 control in your app are similar in functionality, but they suit different needs depending on the app requirements, as follows:
 
 | Approach | Description | Optimized for |
 |---|---|---|
 | Windowed hosting | The WebView2 control takes input from the operating system (OS).  The OS sends the input to the WebView2. | Displaying web content quickly and easily, without having to include features for inputs, outputs, and accessibility. |
+| Window to Visual hosting | A combination of Windowed and Visual hosting. Similar to Windowed hosting except that WebView2 content is output to a visual that is hosted in a window rather having content output to the window directly. | A developer experience nearly identical to Windowed hosting, but with improved DPI/scaling handling and the caveat that the Windows Shell Handwriting experience is unsupported. |
```

Comment:
Windows Shell Handwriting: link to these webpages?
https://learn.microsoft.com/windows/win32/input_ink/input-ink-portal
https://learn.microsoft.com/windows/win32/api/shellhandwriting/
 
Search for articles at LMC:
https://learn.microsoft.com/en-us/search/?terms=Shell%20Handwriting